OutputResults/OutputResults.c

```diff
@@ -0,0 +1,40 @@
+#include "OutputResults.h"
+
+int outputPoints(char* filename) {
+    FILE* fp;
+    struct point* point;
+
+    // check for valid filename
+    if (filename == NULL) {
+        #ifdef DEBUG
+            printf("ERROR null filename\n");
+        #endif
+        return 0;
+    }
+    
+    // open file in write mode
+    if ((fp = fopen(filename, "w")) == NULL) {
+        #ifdef DEBUG
+            printf("ERROR opening file\n");
+        #endif
+        return 0;
+    }
+
+    // output points to file
+    int i = 0;
+    point = getPoint(0);
+    while (point != NULL) {
+        fprintf (
+            fp, "%f %f %f\n", 
```

Comment:
in-class the prof said this should probably not just be the numbers and suggested maybe a csv file with headings. ill approve the changes but work with group 1 to figure out a clean way to store this information and please document it in the readMe for the prof.